[2024-01-14T15:08:41.043+0200] {processor.py:161} INFO - Started process (PID=16234) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:08:41.044+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:08:41.045+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:08:41.045+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:08:41.048+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:08:41.048+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:08:41.049+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:08:41.048+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:08:41.049+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:08:41.058+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-01-14T15:09:29.972+0200] {processor.py:161} INFO - Started process (PID=16548) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:09:29.974+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:09:29.975+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:09:29.975+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:09:29.978+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:09:29.978+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:09:29.979+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:09:29.978+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:09:29.979+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:09:29.988+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-01-14T15:10:27.576+0200] {processor.py:161} INFO - Started process (PID=16786) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:10:27.577+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:10:27.579+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:10:27.578+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:10:27.583+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:10:27.583+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:10:27.584+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:10:27.584+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:10:27.584+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:10:27.597+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-01-14T15:11:25.323+0200] {processor.py:161} INFO - Started process (PID=17015) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:11:25.325+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:11:25.326+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:11:25.325+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:11:25.329+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:11:25.329+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:11:25.329+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:11:25.329+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:11:25.330+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:11:25.340+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-01-14T15:12:20.357+0200] {processor.py:161} INFO - Started process (PID=17270) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:12:20.359+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:12:20.360+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:12:20.360+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:12:20.365+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:12:20.365+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:12:20.365+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:12:20.365+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:12:20.366+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:12:20.377+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-01-14T15:13:14.090+0200] {processor.py:161} INFO - Started process (PID=17507) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:13:14.091+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:13:14.093+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:13:14.092+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:13:14.097+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:13:14.096+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:13:14.098+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:13:14.097+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:13:14.098+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:13:14.110+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-01-14T15:20:27.368+0200] {processor.py:161} INFO - Started process (PID=19567) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:20:27.370+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:20:27.371+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:20:27.370+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:20:27.374+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:20:27.374+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:20:27.375+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:20:27.374+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:20:27.375+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:20:27.386+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-01-14T15:21:17.124+0200] {processor.py:161} INFO - Started process (PID=19760) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:21:17.127+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:21:17.129+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:21:17.128+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:21:17.133+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:21:17.132+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:21:17.133+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:21:17.133+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:21:17.134+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:21:17.145+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-01-14T15:22:13.861+0200] {processor.py:161} INFO - Started process (PID=19987) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:22:13.862+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:22:13.863+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:22:13.863+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:22:13.867+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:22:13.866+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:22:13.867+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:22:13.867+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:22:13.867+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:22:13.877+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-01-14T15:38:20.178+0200] {processor.py:161} INFO - Started process (PID=24690) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:38:20.179+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:38:20.180+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:38:20.180+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:38:20.183+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:38:20.182+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:38:20.183+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:38:20.183+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:38:20.183+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:38:20.192+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-01-14T15:39:08.380+0200] {processor.py:161} INFO - Started process (PID=25090) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:39:08.382+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:39:08.383+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:39:08.382+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:39:08.385+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:39:08.385+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:39:08.385+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:39:08.385+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:39:08.386+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:39:08.396+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-01-14T15:39:57.815+0200] {processor.py:161} INFO - Started process (PID=25287) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:39:57.817+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:39:57.818+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:39:57.818+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:39:57.823+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:39:57.822+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:39:57.823+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:39:57.823+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:39:57.824+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:39:57.837+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-01-14T15:40:55.149+0200] {processor.py:161} INFO - Started process (PID=25512) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:40:55.150+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:40:55.150+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:40:55.150+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:40:55.153+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:40:55.152+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:40:55.153+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:40:55.153+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:40:55.153+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:40:55.161+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-01-14T15:41:45.852+0200] {processor.py:161} INFO - Started process (PID=25772) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:41:45.854+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:41:45.856+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:41:45.855+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:41:45.861+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:41:45.860+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:41:45.861+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:41:45.861+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:41:45.861+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:41:45.871+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-01-14T15:42:44.943+0200] {processor.py:161} INFO - Started process (PID=26192) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:42:44.944+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:42:44.945+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:42:44.945+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:42:44.948+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:42:44.948+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:42:44.948+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:42:44.948+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:42:44.949+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:42:44.958+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-01-14T15:43:52.775+0200] {processor.py:161} INFO - Started process (PID=26836) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:43:52.777+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:43:52.777+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:43:52.777+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:43:52.780+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:43:52.779+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:43:52.780+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:43:52.780+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:43:52.780+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:43:52.794+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-01-14T15:45:06.830+0200] {processor.py:161} INFO - Started process (PID=27710) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:45:06.831+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:45:06.832+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:45:06.832+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:45:06.834+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:45:06.834+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:45:06.834+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:45:06.834+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:45:06.834+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:45:06.853+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-01-14T15:45:58.369+0200] {processor.py:161} INFO - Started process (PID=27914) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:45:58.372+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:45:58.372+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:45:58.372+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:45:58.377+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:45:58.376+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:45:58.377+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:45:58.377+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:45:58.377+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:45:58.387+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-01-14T15:46:52.638+0200] {processor.py:161} INFO - Started process (PID=28126) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:46:52.640+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:46:52.642+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:46:52.641+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:46:52.646+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:46:52.645+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:46:52.646+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:46:52.646+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:46:52.647+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:46:52.656+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-01-14T15:47:45.340+0200] {processor.py:161} INFO - Started process (PID=28430) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:47:45.342+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:47:45.343+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:47:45.343+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:47:45.346+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:47:45.346+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:47:45.346+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:47:45.346+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:47:45.347+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:47:45.356+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-01-14T15:48:38.451+0200] {processor.py:161} INFO - Started process (PID=28640) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:48:38.453+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:48:38.454+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:48:38.454+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:48:38.457+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:48:38.457+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:48:38.457+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:48:38.457+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:48:38.458+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:48:38.467+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-01-14T15:49:33.724+0200] {processor.py:161} INFO - Started process (PID=28852) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:49:33.726+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:49:33.727+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:49:33.727+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:49:33.730+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:49:33.730+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:49:33.731+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:49:33.731+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:49:33.731+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:49:33.741+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2024-01-14T15:50:23.251+0200] {processor.py:161} INFO - Started process (PID=29050) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:50:23.252+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:50:23.254+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:50:23.253+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:50:23.258+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:50:23.257+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:50:23.258+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:50:23.258+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:50:23.258+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:50:23.270+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-01-14T15:51:16.188+0200] {processor.py:161} INFO - Started process (PID=29260) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:51:16.189+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:51:16.190+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:51:16.190+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:51:16.193+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:51:16.193+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:51:16.194+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:51:16.194+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:51:16.194+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:51:16.204+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-01-14T15:52:08.910+0200] {processor.py:161} INFO - Started process (PID=29471) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:52:08.912+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:52:08.913+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:52:08.913+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:52:08.916+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:52:08.915+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:52:08.916+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:52:08.916+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:52:08.916+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:52:08.925+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-01-14T15:52:57.993+0200] {processor.py:161} INFO - Started process (PID=29695) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:52:57.995+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:52:57.995+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:52:57.995+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:52:57.999+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:52:57.998+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:52:57.999+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:52:57.999+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:52:57.999+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:52:58.008+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-01-14T15:53:52.621+0200] {processor.py:161} INFO - Started process (PID=29908) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:53:52.625+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:53:52.628+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:53:52.627+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:53:52.641+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:53:52.639+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:53:52.642+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:53:52.641+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:53:52.642+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:53:52.680+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.071 seconds
[2024-01-14T15:54:56.040+0200] {processor.py:161} INFO - Started process (PID=30122) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:54:56.047+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:54:56.052+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:54:56.050+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:54:56.070+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:54:56.067+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:54:56.071+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:54:56.071+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:54:56.073+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:54:56.133+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.111 seconds
[2024-01-14T15:56:02.474+0200] {processor.py:161} INFO - Started process (PID=30349) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:56:02.479+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:56:02.482+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:56:02.481+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:56:02.495+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:56:02.493+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:56:02.496+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:56:02.495+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:56:02.497+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:56:02.537+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.075 seconds
[2024-01-14T15:57:00.529+0200] {processor.py:161} INFO - Started process (PID=30560) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:57:00.533+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:57:00.536+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:57:00.535+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:57:00.547+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:57:00.546+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:57:00.548+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:57:00.548+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:57:00.549+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:57:00.586+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-01-14T15:57:53.170+0200] {processor.py:161} INFO - Started process (PID=30779) to work on /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:57:53.174+0200] {processor.py:830} INFO - Processing file /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-14T15:57:53.176+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:57:53.175+0200] {dagbag.py:538} INFO - Filling up the DagBag from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:57:53.182+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:57:53.181+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-14T15:57:53.183+0200] {logging_mixin.py:188} INFO - [2024-01-14T15:57:53.183+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-14T15:57:53.183+0200] {processor.py:842} WARNING - No viable dags retrieved from /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-14T15:57:53.198+0200] {processor.py:183} INFO - Processing /home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
